HelpSource/Guides/Debugging-tips.schelp

```diff
@@ -60,39 +60,45 @@ l.flat.plot(numChannels: 2);
 
 subsection:: Polling
 
-Polling uses a UGen to have the server output values itself.
-
-The simplest way is to use the .poll method, which will output your value 10 times per second.
+Polling allows you to debug a SynthDef by printing samples of a UGen's output to the post window. To do this, use the .poll method (a shorthand for the Poll UGen), which prints 10 times per second by default.
```

Comment:
better to use use the inline `code::.poll::` formatting here; if you did this it could be more concisely phrased as "To do this, use `.poll` (a shorthand"